From 686bbcd00eb4aa8f55a68a963dd1cb93af7ed8b0 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:40 -0400
Subject: [redhat] kabi: Adding symbol blk_init_queue (block/blk-core.c)

Message-id: <50f0ef2ae8e4a8e5da08f09405cb56ab54dda57c.1410510773.git.poros@redhat.com>
Patchwork-id: 95971
O-Subject: [PATCH RHEL7.1 040/233] kabi: Adding symbol blk_init_queue (block/blk-core.c)
Bugzilla: 1062361 1071097
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361,1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Prepare a request queue for use with a block device.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/blk_init_queue
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/blk_init_queue
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/blk_init_queue

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/blk_init_queue b/redhat/kabi/kabi-module/kabi_ppc64/blk_init_queue
new file mode 100644
index 0000000..870c8e8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/blk_init_queue
@@ -0,0 +1,2 @@
+#1-
+0x9846b739 blk_init_queue vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/blk_init_queue b/redhat/kabi/kabi-module/kabi_s390x/blk_init_queue
new file mode 100644
index 0000000..951db7b
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/blk_init_queue
@@ -0,0 +1,2 @@
+#1-
+0xf77e96a1 blk_init_queue vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/blk_init_queue b/redhat/kabi/kabi-module/kabi_x86_64/blk_init_queue
new file mode 100644
index 0000000..c26ac13
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/blk_init_queue
@@ -0,0 +1,2 @@
+#1-
+0x84daafd0 blk_init_queue vmlinux EXPORT_SYMBOL
-- 
1.7.1